<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>789</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1172</width>
    <height>789</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1172</width>
    <height>789</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow, QWidget {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #0f2027, stop:0.5 #203a43, stop:1 #2c5364);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_search">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    background-color: white;
    color: #2c5364;           /* Tema rengine zıt ve okunabilir */
    border: 2px solid #203a43;
    border-radius: 5px;
    padding: 5px;
    font-size: 12pt;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_search">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #8f8f8f;
    color: white;
    border-radius: 8px;
    padding: 6px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #cc0000;
}
</string>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../images/search.png</normaloff>../images/search.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_all_submitted_projects">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #a40000;
    color: white;
    border: none;
    border-radius: 8px;
    padding: 8px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #cc0000;
}
</string>
        </property>
        <property name="text">
         <string>Submitted Projects</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_project_arrivals">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #a40000;
    color: white;
    border: none;
    border-radius: 8px;
    padding: 8px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #cc0000;
}
</string>
        </property>
        <property name="text">
         <string>Project Arrivals</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_BackMenu">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #a40000;
    color: white;
    border: none;
    border-radius: 8px;
    padding: 8px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #cc0000;
}
</string>
        </property>
        <property name="text">
         <string>Back Menu</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../images/back.png</normaloff>../images/back.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_exit">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #a40000;
    color: white;
    border: none;
    border-radius: 8px;
    padding: 8px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #cc0000;
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../images/logout.png</normaloff>../images/logout.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="4">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="7">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="1" colspan="7">
     <widget class="QTableWidget" name="tableWidget">
      <property name="styleSheet">
       <string notr="true">QHeaderView::section {
    background-color: #2c5364;
    color: white;
    padding: 4px;
    font-weight: bold;
    border: 1px solid #203a43;
}

QTableWidget {
    gridline-color: white;
    color: white;  /* ← Bu satır metinleri beyaz yapar */
}</string>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>155</number>
      </attribute>
      <column>
       <property name="text">
        <string>Name Surname</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Project Sending Date</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Project Arrival Date</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="5">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="7">
     <widget class="QComboBox" name="comboBox_language">
      <property name="styleSheet">
       <string notr="true">QComboBox#comboBox_language {
    background-color: #ffffff;
    color: #2c5364;  /* Tema ile uyumlu koyu mavi */
    border: 2px solid #2c5364;
    border-radius: 5px;
    padding: 4px;
    min-width: 10px;
    font-weight: bold;
}
QComboBox QAbstractItemView {
    background-color: #ffffff;
    color: #2c5364;
    selection-background-color: #203a43;
    selection-color: white;
}</string>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Türkçe</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>English</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dutch</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Français</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Español</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Deutsch</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Русский </string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>العربية </string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>简体中文</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="6">
     <widget class="QLabel" name="label_language">
      <property name="maximumSize">
       <size>
        <width>65</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#label_language {
    color: white;
    font-weight: bold;
    font-size: 6pt;
}
</string>
      </property>
      <property name="text">
       <string>Languages</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_logo">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../images/logo.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_interviews_menu">
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    font-family: &quot;Georgia&quot;;
    font-size: 18pt;
    color: white;
}
</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;We R Here Interviews Menu&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1172</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
